### ISSUE TITLE
Segmentation for one given video

### ISSUE THREAD

**Issue by MAGomes95** [echonet/dynamic#63]
Hello,

When training the segmentation model it only outputs and saved segmentation videos associated with the test set. I am wondering if it is possible, given the trained model, to produce and save segmentations videos of the videos associated to the train and validation set.

Thank you,

**Comment by LitMSCTBB** [comment]
You can just use InitializationNotebook.ipynb. You'd make a new video directory with just the train/val videos and in the notebook you specify the directory you stored them in. Also make sure you specify the right locations for the weights (the .pt model files, either the inference models found on the release page for this repo, or the model found in the output of your own train/test process).
